src/modules/pages/components/ExpenditurePage/ExpenditureTemplate.tsx

```diff
@@ -0,0 +1,26 @@
+import React, { ReactNode } from 'react';
```

Comment:
Didn't notice it the first time but I think that this file should be called `ExpenditurePage`. So the file name corresponds to the folder where it is. Remember to update the display name.